api/src/test/resources/org/openmrs/module/xstream/include/PersonSerializationTest.xml

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <dataset>
-    <person person_id="1000" gender="M" birthdate="1945-12-30" birthdate_estimated="0" dead="1" death_date="2005-02-10 15:30:00.0" cause_of_death="1088" creator="1" date_created="2006-01-18 00:00:00.0" voided="1" voided_by="1" date_voided="2006-09-18 00:00:00.0" void_reason="test purpose" uuid="04079813-4c9d-4f9d-b676-4c0502a5c1c3"/>
-    <concept concept_id="1088" retired="0" datatype_id="2" class_id="7" is_set="0" creator="1" date_created="2005-01-12 00:00:00.0" changed_by="1" date_changed="2006-02-14 14:14:22.0" uuid="001ba2b2-92b2-102c-adee-6014420f8468"/>
-    <concept_datatype concept_datatype_id="2" name="Coded" hl7_abbreviation="CWE" description="Value determined by term dictionary lookup (i.e., term identifier)" creator="1" date_created="2004-02-02 00:00:00.0" retired="0" uuid="003d8f84-92b2-102c-adee-6014420f8468"/>
```

Comment:
Was this simply formatting? It is hard to differentiate between the actual changes and formatting.